intranet/static/js/schedule.js

```diff
@@ -31,29 +30,29 @@ $(document).ready(function() {
 
     scheduleView = function(reldate) {
         $sch = $(".schedule");
+
         var endpoint = $sch.attr("data-endpoint");
         var prev = $sch.attr("data-prev-date");
-        var next = $sch.attr("data-next-date")
-        if (reldate === 1) {
-            date = next;
-        } else if (reldate === -1) {
-            date = prev;
-        } else {
-            date = reldate;
-        }
+        var next = $sch.attr("data-next-date");
+
+        if (reldate === 1) date = next;
+        else if (reldate === -1) date = prev;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["date" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2747126376/issues/source?bid=3212528&fileBranchId=3355681#l39)